src/test/java/com/google/api/codegen/testdata/ruby_smoke_test_library.baseline

```diff
@@ -0,0 +1,41 @@
+============== file: test/library/v1/library_service_smoke_test.rb ==============
```

Comment:
We could do this, but based on current structure of XXXGapicTestSurfaceTransformer, unit test and smoke test were generated together hence they are put into the same place. We can differentiate them based on their postfix such as library_service_smoke_test.rb and libray_service_unit_test.rb